tests/fixtures.py

```diff
@@ -35,7 +35,7 @@ def test_base_dir():
     yield directory
 
     if os.listdir(directory) == []:
-        shutil.rmtree(directory)
+        pass #shutil.rmtree(directory)
```

Comment:
This are probably not what you want here.

Reply:
We could add an env variable that skips the cleanup if you'd like

Reply:
We might need to.  I think I had to do this to stop cleanup for some valgrind errors at one point...